app/controllers/issues_controller.rb

```diff
@@ -0,0 +1,26 @@
+class IssuesController < ApplicationController
+  skip_before_action :authenticate_user!, only: [:new, :create]
+  def new
+    @issue = Issue.new
+    authorize(@issue)
+  end
+
+  def create
+    @issue = Issue.new(issues_params)
+    authorize(@issue)
+    @issue.user_id = current_user.id
```

Comment:
You use skip_before_action :authenticate_user! so you will not have current_user.
Also you have condition to check current_user presence in policy, this is not OK. Policy is place where you check users ability to do something but not existence of that user.